<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <comment>
 * Released as open source by Gabriel Caudrelier
 *
 * Developed by Gabriel Caudrelier, gabriel dot caudrelier at gmail dot com
 *
 * https://github.com/metrodango/pip3line
 *
 * Released under BSD 3-Clause see LICENSE for more information
</comment>
 <class>TransformDisplayConf</class>
 <widget class="QWidget" name="TransformDisplayConf">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <item>
    <widget class="QWidget" name="nameWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="nameLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="nameLineEdit"/>
      </item>
      <item>
       <widget class="QComboBox" name="transformComboBox"/>
      </item>
      <item>
       <widget class="QPushButton" name="infoPushButton">
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Help related t the selected transform</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../gui_res.qrc">
          <normaloff>:/Images/icons/document-properties.png</normaloff>:/Images/icons/document-properties.png</iconset>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="readonlyCheckBox">
     <property name="text">
      <string>Read only</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="generalWidget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QGroupBox" name="wayGroupBox">
        <property name="title">
         <string>Direction</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>10</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item alignment="Qt::AlignLeft">
          <widget class="QRadioButton" name="inboundRadioButton">
           <property name="text">
            <string>Inbound </string>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignLeft">
          <widget class="QRadioButton" name="outboundRadioButton">
           <property name="text">
            <string>Outbound</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="formatGroupBox">
        <property name="title">
         <string>Output format</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>10</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item alignment="Qt::AlignLeft">
          <widget class="QRadioButton" name="hexaRadioButton">
           <property name="text">
            <string>He&amp;xadecimal</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="textRadioButton">
           <property name="text">
            <string>Text (UTF-&amp;8)</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="typeGroupBox">
        <property name="title">
         <string>Display type</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QRadioButton" name="oneLineRadioButton">
           <property name="text">
            <string>One &amp;line</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="blockRadioButton">
           <property name="text">
            <string>Block (&amp;multi-lines)</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../gui_res.qrc"/>
 </resources>
 <connections/>
</ui>
